depthai_nodes/parser_generator.py

```diff
@@ -53,7 +54,23 @@ def build(self, nn_archive: dai.NNArchive, head_index: int = None) -> Dict:
                 )
 
             head = decode_head(head)
-            parsers[index] = pipeline.create(parser).build(head)
+            sig = inspect.signature(parser.build)
+            if "inputs_size" in sig.parameters:
+                inputs_size = []
+                for input in nn_archive.getConfig().model.inputs:
+                    breakpoint()
```

Comment:
Probably needs to be removed?

Reply:
Fixed in [1ed14ba](https://github.com/luxonis/depthai-nodes/pull/126/commits/1ed14baf5553f595cd0dc1f46359552579674f7b).